From 8dc9ed67fab10b201446dc4b762482f6e018b1f2 Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Thu, 3 Jun 2021 08:20:40 -0700
Subject: [PATCH 114/196] Ensure rpc_generateblock always passes (#191)

While redenominating the code to 10^6 from 10^8, this particular
spot was missed. The test continued to pass because `listunspent` is
nondeterministic. Sometimes it was pulling a tx with sufficient funds,
and sometimes it was creating transactions that overspent. This change
ensures that both transactions that can be used in subsequent lines
have enough funds to pass the test.
---
 test/functional/rpc_generateblock.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/functional/rpc_generateblock.py b/test/functional/rpc_generateblock.py
index e60bc3616..04df9bac7 100755
--- a/test/functional/rpc_generateblock.py
+++ b/test/functional/rpc_generateblock.py
@@ -66,7 +66,7 @@ class GenerateBlockTest(BitcoinTestFramework):
             node.sendtoaddress(address, 0.1)
 
         self.log.info('Generate block with txid')
-        txid = node.sendtoaddress(address, 1)
+        txid = node.sendtoaddress(address, 100)
         hash = node.generateblock(address, [txid])['hash']
         block = node.getblock(hash, 1)
         assert_equal(len(block['tx']), 2)
-- 
2.39.5 (Apple Git-154)

